### ISSUE TITLE
conductor server is failed to run because of wrong jar file name

### ISSUE THREAD

**Issue by komatiraju032** [Netflix/conductor#2199]
I have started the conductor using `docker-compose up -d`. I'm getting the below error for conductor-server
```
Starting Conductor server

Property file: config.properties

config.properties

Log4j file:


Using 'config.properties'

Using java options config:

Error: Unable to access jarfile conductor-server-*-all.jar
```

**Comment by komatiraju032** [comment]
I fixed the issue by changing "all" with "boot"(conductor-server-*-boot.jar) in `docker/server/bin/startup.sh` file

**Comment by maheshyaddanapudi** [comment]
Hi @komatiraju032 

I am raising a PR with this and few other docker compose big fixes.

Will post the PR# here once done.

**Comment by maheshyaddanapudi** [comment]
Raised PR# https://github.com/Netflix/conductor/pull/2206

**Comment by aravindanr** [comment]
This bug is addressed in [v3.0.3](https://github.com/Netflix/conductor/releases/tag/v3.0.3)
